composer.json

```diff
@@ -4,12 +4,11 @@
         "php": "^7.0"
     },
     "require-dev": {
-        "phpspec/phpspec": "^3.4",
-        "leanphp/phpspec-code-coverage": "^3.1"
+        "phpunit/phpunit": "^6.3"
     },
     "autoload": {
         "psr-4": {
-            "Craftwork\\": "src/"
+            "Craftwork\\": "src/Craftwork"
```

Comment:
I'm curious why you've done this. I generally prefer the other way and in fact I would suggest going `"Craftwork\\IdObfuscator\\": "src/"` to get rid of some redundant folders.

Also if you don't want to do that you can set this to `""": "src/"` instead